### DISCUSSION TITLE
不知道为什么一直在给我发邮箱,我就在一个帖子下回了一个消息

### DISCUSSION THREAD

**Discussion by Onlyonehite** [share121/inter-knot#2814] [其他]
![微信图片_20240818213528](https://github.com/user-attachments/assets/211681f9-2e54-4e85-9991-05b8da620014)
你们有遇到吗,我该怎么办

**Comment by HKLov3** [comment]
![image](https://github.com/user-attachments/assets/b8eb7187-34f1-4ad0-bc99-9af01c08c0f6)
进入个人主页，点击设置，点击通知选项，如图，取消电子邮件通知的勾选，这样邮箱就不会再发送邮件了

**Reply by Onlyonehite** [reply]
谢谢大佬,已经改好了<img src="https://img-static.mihoyo.com/communityweb/upload/baaaf420abb173dac46a466b5115e3ce.gif" alt="米游姬 感谢" width="75" >
